webpack/optimization/minimizer.js

```diff
@@ -1,5 +1,5 @@
 const TerserPlugin = require( 'terser-webpack-plugin' );
-const OptimizeCSSAssetsPlugin = require( 'optimize-css-assets-webpack-plugin' );
+const CssMinimizerPlugin = require( 'css-minimizer-webpack-plugin' );
```

Comment:
`optimize-css-assets-webpack-plugin` is not recommended for webpack 5: https://www.npmjs.com/package/optimize-css-assets-webpack-plugin